src/main/target/CC3D/target.h

```diff
@@ -126,9 +124,15 @@
 #define SONAR_EXTI_PIN_SOURCE       GPIO_PinSource0
 #define SONAR_EXTI_IRQN             EXTI0_IRQn
 
-#define USE_SERIAL_4WAY_BLHELI_INTERFACE
+#define GPS
+#define BLACKBOX
+#define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
+
+#define TELEMETRY
+#define SERIAL_RX
+#define USE_SERVOS
+#define USE_CLI
+#define TARGET_MOTOR_COUNT 6
+
+
```

Comment:
just a note to say that re-arranging code as part of a PR makes it harder to review the changes.  If cleanups can be done in different PR's that'd be help for speedier code reviews for functional changes.